js_modules/dagit/packages/core/src/asset-graph/AssetNode.tsx

```diff
@@ -131,22 +157,53 @@ export const AssetNodeStatusRow: React.FC<{
   const late = isAssetLate(liveData);
 
   if (materializingRunId) {
-    return (
-      <AssetNodeStatusBox background={Colors.Blue50}>
-        <AssetLatestRunSpinner liveData={liveData} />
-        <Caption style={{flex: 1}} color={Colors.Gray800}>
-          Materializing...
-        </Caption>
-        <AssetRunLink runId={materializingRunId} />
-      </AssetNodeStatusBox>
-    );
+    return {
+      background: Colors.Blue50,
+      border: Colors.Blue500,
+      content: (
+        <>
+          <AssetLatestRunSpinner liveData={liveData} />
+          <Caption style={{flex: 1}} color={Colors.Gray800}>
+            Materializing...
+          </Caption>
+          <AssetRunLink runId={materializingRunId} />
+        </>
+      ),
+    };
+  }
+
+  if (liveData.partitionStats) {
+    const {numPartitions, numMaterialized} = liveData.partitionStats;
+    const numMissing = numPartitions - numMaterialized;
+    return {
+      background: late ? Colors.Red50 : numMissing ? Colors.Yellow50 : Colors.Green50,
+      border: late ? Colors.Red500 : numMissing ? Colors.Yellow500 : Colors.Green500,
+      content: (
+        <>
+          <Caption color={late ? Colors.Red700 : numMissing ? Colors.Yellow700 : Colors.Green700}>
+            {`${numPartitions.toLocaleString()} Partitions`}
```

Comment:
- lowercase `partitions`
- Do we need a singular `1 partition` case?

Reply:
Hmm good call re: 1 partition case -- it does feel a bit odd to uppercase it, I'll run this by josh